.goreleaser.yaml

```diff
@@ -0,0 +1,73 @@
+env:
+- GOPROXY=https://proxy.golang.org|direct
+- GO111MODULE=on
+- CGO_ENABLED=0
+- PKG=github.com/operator-framework/combo/pkg/version
+- KUBERNETES_VERSION=v0.22.2
```

Comment:
This feels potentially brittle. Can we try and use the Makefile as the source of truth here, and export any variable that sets the Kubernetes version (if that variable even exists today)?

Reply:
Yup that's more-or-less what I had in mind.

Reply:
I think so. I think one of the other variables is exported that way already.